src/data/javascript-quiz.ts

```diff
@@ -1968,6 +1968,18 @@ const javascriptQuiz = [
     Explanation:
       "An IIFE or Immediately Invoked Function Expression, is used to declare a private variable and to avoid polluting the global environment. If a function is declared with an IIFE by using a parenthesis around it, the variable declared inside it becomes private and its value can be accessed using a getter function inside the original function.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/IIFE"
+  },
+  {
+    Question:
+      "In JavaScript, which of the following statements is correct regarding async await?",
+    Answer: "`async` automatically returns a promise",
+    Distractor1: "`await` keeps executing code regardless of promies status",
+    Distractor2: "`async` can be used independently without an async function",
+    Distractor3:
+      "Multiple `await` keywords can be used in a single line inside an `async` function",
+    Explanation:
+      "An `async function` declaration creates an [`AsyncFunction`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AsyncFunction) object. Each time when an async function is called, it returns a new [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) which will be resolved with the value returned by the async function, or rejected with an exception uncaught within the async function.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function"
   }
```

Comment:
please remove all code block syntax like backticks `` and markdown syntax like `[]()` because it does not render well in the final quiz as evident in these screenshots here
<img width="855" alt="Screenshot 2023-10-16 at 4 07 52 PM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/58e1b5dc-412e-40a0-bfa1-604b7c59526c">
<img width="539" alt="Screenshot 2023-10-16 at 4 08 09 PM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/445ae9d2-96d9-4de4-bce5-9c7d7059e116">

Reply:
apologies for the oversight, just fixed the markdowns and recommitted!